README.md

```diff
@@ -1,11 +1,283 @@
+
+There are several technical advantages to using the native OMSDK over a web player-based solution. We won’t go into these in depth in this document, but here are some of the advantages:
+* Performance
+* Mobile network awareness
+* Frugal memory, thread, and device resource usage
+* Security – webviews / embedded browsers are known weak points
+* Fine-grained control, less limits
+* Some platforms don’t have webviews (e.g., Apple TV)
+* More customization options
+
+## Advertising Info and User Tracking <a name="privacy"></a>
+The O2 Mobile SDK does not track anything that is not related to playing videos or video ads. We use the IDFA (ID for advertisers) value and respect the user's settings for Limit Ad Tracking (iOS enforces this anyway). The device geolocation is determined by our backend video servers based on IP address, for the purposes of determining and filtering out content that is geo-restricted by content owners. The SDK does not explicitly use the built-in Location Services APIs, and thus does not require your users to grant access to device location data.
+
+## Starting Requirements <a name="requirements"></a>
+* **Apple Xcode 8+**
+* **Swift 3.1 or 3.2***
+* **CocoaPods or Carthage**
+* **Mobile device running iOS 8 or later or AppleTV device running tvOS 9 or later**
+* **Account in the O2 Portal, and access to Oath-ingested video content**
+* **Onboarded application bundle ID**
+
+**\* Note**: Swift 4 support is coming early in Q1 of 2018.
+
+## Onboarding your Apps for SDK Authentication <a name="onboarding"></a>
+In order for the OMSDK to authenticate itself for video playback within your app, we pass the containing app’s unique App Store bundle identifier to our back end service. You need to email the [Video Support Team](mailto:video.support@oath.com) to register all of your app bundle IDs for OMSDK usage. You can also register multiple bundle IDs against your same app entity. Possible reasons to do this, is to allow for a dev/test app bundle ID or an enterprise bundle ID, that can co-exist on a device alongside your production app. Also, both iOS and Android app bundle IDs can either be the same or different – for the same app. Registration not only authenticates your application, but it ensures your backend video and ads analytics are all configured properly. In addition, this registration information also defines all the video content your app is allowed to playback through the SDK.
+
+The sample projects are all set up to use the following test-only bundle ID: `com.aol.mobile.one.testapp`
+
+## High-Level Architecture Overview <a name="architecture"></a>
+At a high-level, the OMSDK architecture is composed of the following components:
+* SDK Core
+* A set of Video Renderers that are either built-in (e.g., flat 2D and RYOT 360°) or external plugins (e.g., Verizon Envrmnt 360°, custom, etc.)
+* O2 VRM (video rights management) VAST Ads Engine
```

Comment:
We don't have this in SDK.

Reply:
what is do we have?  VAST ads support to the o2 VRM standard.